frontend/src/container/SideNav/styles.ts

```diff
@@ -24,3 +24,15 @@ export const Sider = styled(SiderComponent)`
 		color: white;
 	}
 `;
+
+interface DarkModeProps {
+	checked?: boolean;
+	defaultChecked?: boolean;
+}
+
+export const ToggleButton = styled(Switch)<DarkModeProps>`
+	&&& {
+		background: ${({checked})=> (checked === false ? 'grey': ' ')};
```

Comment:
if checked is true 

can we use `unset` ?

Reply:
yes, as if it is true then default colour of antd component will show up

Reply:
![image](https://user-images.githubusercontent.com/59202075/146955095-6c372e2c-2e11-4e05-8e50-6eb29e0400d3.png)

like this